Pod/Classes/LSTracer.m

```diff
@@ -8,7 +8,7 @@
 #import "LSUtil.h"
 #import "LSVersion.h"
 
-NSString* const LSDefaultHostport = @"collector.lightstep.com:443";
+NSString* const LSDefaultHostport = @"collector.lightstep.com:443/api/v0/reports";
```

Comment:
... and we should change the comments and internal variable name accordingly, too.